<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileRunner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="hostPath">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>26</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="remotePath">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>441</width>
      <height>26</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="remoteStatus">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>441</width>
      <height>101</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="hostQueue">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>441</width>
      <height>101</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time remaining</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Path</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Site</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="hostTree">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>441</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="remoteTree">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>441</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <zorder>remoteStatus</zorder>
   <zorder>hostPath</zorder>
   <zorder>remotePath</zorder>
   <zorder>hostQueue</zorder>
   <zorder>hostTree</zorder>
   <zorder>remoteTree</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Connect"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Preferences"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Connect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
